internal/tui/components/chat/splash/splash.go

```diff
@@ -263,7 +263,9 @@ func (s *splashCmp) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			}
 		case key.Matches(msg, s.keyMap.Yes):
 			if s.isOnboarding {
-				return s, nil
+				u, cmd := s.modelList.Update(msg)
+				s.modelList = u
+				return s, cmd
 			}
 			if s.needsAPIKey {
 				u, cmd := s.apiKeyInput.Update(msg)
```

Comment:
Hey @kujtimiihoxha, in this branch the fix was only applied to the model list. If you try to press Y or N on the API key field, it still doesn't work.

We need to move the `if s.isOnboarding {` to be after the `if s.needsAPIKey {` one, which I have done on #402.